### PR TITLE
CLI variable command changes

### PR DESCRIPTION
**Pull request by Bowrna**
closes: https://github.com/apache/airflow/issues/22708
related: https://github.com/apache/airflow/issues/22708

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

File: airflow/cli/commands/variable.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    'files',
+    type=click.Path(exists=True),
+)
+
+@airflow_cmd.group()
+@click.pass_context
+def variable(ctx):
+    pass
+
+@suppress_logs_and_warning
```
Comment (hankehly) [inline airflow/cli/commands/variable.py:62]: Hi @Bowrna 

How do you think to handle `suppress_logs_and_warning` ? It expects a `verbose` flag as the first function argument.

Comment (hankehly) [reply airflow/cli/commands/variable.py:62]: I'm thinking about doing something like this:

```py
@airflow_cmd.command("foo")
@click_verbose
@suppress_logs_and_warning_click_compatible
def cheat_sheet(verbose):
    pass
```
```py
def suppress_logs_and_warning_click_compatible(f: T) -> T:
    @functools.wraps(f)
    def _wrapper(*args, **kwargs):
        if kwargs.get("verbose"):
            f(*args, **kwargs)
        else:
            ... do the same as `suppress_logs_and_warning`
```

Comment (Bowrna) [reply airflow/cli/commands/variable.py:62]: ah thanks for pointing me to that part @hankehly. I will check yours for more details

**Comment by Bowrna** [comment]
@blag I made the changes for the variable command. Share your reviews on this PR when you get time. Thanks.

**Comment by blag** [comment]
Thank you for all of the work you've done for this.

From [this discussion](https://github.com/apache/airflow/pull/24591#pullrequestreview-1019268731), there is more groundwork and research necessary before we can evaluate rewriting the CLI with Click.

**Pull request closed without merging**